jangaroo-maven/jangaroo-maven-plugin/src/main/java/net/jangaroo/jooc/mvnplugin/AbstractCompilerMojo.java

```diff
@@ -292,6 +301,8 @@ public void warning(String msg) {
 
     int result = compile(jooc);
 
+    printDependencyWarnings(configuration);
+    Paths.get(configuration.getWarningsOutputDirectory(), "dependencyWarnings").toFile().delete();
```

Comment:
What if someone (mis)configures `findDependencyErrors`  + no `warningsOutputDirectory` (or the new `dependencyReportOutputfile`, wouldn't that result in a NPE here?

Reply:
I changed it so there are no dependency checks if there is no provided path. I also added a small note about that in the JoocConfiguration to the property.